### PR TITLE
No Mr. Mouse, we don't have any metroids here

### PR DESCRIPTION
**Pull request by francinum**
no siree

Purges the custom items, it's fosstar's job to clean up the icons
yip yip, you little shit.

### CODE REVIEW

**Comment by CosmicScientist** [comment]
` ////SS13 DONATOR CUSTOM ITEM STORAGE ZONE OF MAGICAL HAPPINESS APOPHIS - LAST UPDATE - 14JUN2016`

You might want to remove or update this line. Commit history does tell us when it's updated.

**Comment by CosmicScientist** [comment]
:+1: we need more cringe and degeneracy to replace what was lost.

File: code/modules/cm_marines/Donor_Items.dm [hunk 1]
```diff
@@ -40,389 +41,6 @@
...
-	desc = "Some fancy looking armor, with lots of lights and buttons.  DONOR ITEM"
-	icon_state = "reactive"
-	item_state = "reactive"
-	blood_overlay_type = "armor"
-
-/obj/item/clothing/suit/storage/marine/fluff/SAS
-	name = "Legion Armor"
-	desc = "This armor was custom made to resemble the small growing Legion within the galaxy started by one man slowly making its way to becoming a larger Corperation.  DONOR ITEM."
-	item_state = "ncrjacket"
-	icon_state = "ncrjacket"
```
Comment (LetterN) [inline code/modules/cm_marines/Donor_Items.dm]: technicaly you can have this if this is the f13 one, f13 one is open sauce

Comment (subject217) [reply code/modules/cm_marines/Donor_Items.dm]: If it stays in it should just be an unused icon, not a donor item.

Comment (francinum) [reply code/modules/cm_marines/Donor_Items.dm]: if any of these deserve to be saved, they can be added as loadout items in a separate PR.

**Comment by F-OS** [comment]
@francinum why don't you do it yourself

**Pull request merged**